docs/docs/index.md

```diff
@@ -5,52 +5,54 @@ Welcome to zamba's documentation!
 <div class="embed-responsive embed-responsive-16by9" width=500>
     <iframe width=600 height=340 class="embed-responsive-item" src="https://s3.amazonaws.com/drivendata-public-assets/monkey-vid.mp4" frameborder="0" allowfullscreen=""></iframe>
 </div>
```

Comment:
How is the div above supposed to appear? It looks like it might need some tweaking